sound/soc/intel/boards/sof_sdw.c

```diff
@@ -219,6 +235,15 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 	},
 	{
 		.part_id = 0x711,
+		.version_id = 3,
+		.direction = {true, true},
+		.dai_name = "rt711-sdca-aif1",
+		.init = sof_sdw_rt711_init,
+		.exit = sof_sdw_rt711_exit,
```

Comment:
maybe we should have different init/exit functions for sdca devices., e.g. so that we can use "%s hs:rt711-sdca" for the component string, and have different pins/widgets.